### PR TITLE
Use reusables actions.

### PR DESCRIPTION
**Pull request by terabytesoftw**
| Q             | A
| ------------- | ---
| Is bugfix?    | ❌
| New feature?  | ✔️
| Breaks BC?    | ❌

### CODE REVIEW

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -22,58 +22,10 @@ on:
...
-      - name: Update composer
-        run: composer self-update
-
-      - name: Install dependencies with composer
-        run: composer update --prefer-dist --no-interaction --no-progress --optimize-autoloader --ansi
-
-      - name: Run tests with phpunit
-        run: vendor/bin/phpunit --colors=always
+  phpunit:
+    uses: yiisoft/actions/.github/workflows/phpunit.yml@master
```
Comment (vjik) [inline .github/workflows/build.yml:26]: In `yiisoft/actions` not need run actions. Suggest move them to root (`yiisoft/actions/phpunit.yml`)

Comment (terabytesoftw) [reply .github/workflows/build.yml:26]: Please read the documentation.

Comment (vjik) [reply .github/workflows/build.yml:26]: OK. https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_iduses

**Review by vjik — APPROVED** [review]

**Comment by terabytesoftw** [comment]
Firtts merge https://github.com/yiisoft/actions/pull/3

**Pull request merged**